code/modules/paperwork/pen.dm

```diff
@@ -139,10 +139,32 @@ var/paperwork_library
 /datum/writing_style/pen/New()
 	addReplacement(REG_BBTAG("*"), "<li>")
 	addReplacement(REG_BBTAG("hr"), "<HR>")
-	addReplacement(REG_BBTAG("small"), "<font size = \"1\">")
-	addReplacement(REG_BBTAG("/small"), "</font>")
+	addReplacement(REG_BBTAG("small"), "<span style=\"font-size:15px\">")
+	addReplacement(REG_BBTAG("/small"), "</span>")
+	addReplacement(REG_BBTAG("tiny"), "<span style=\"font-size:10px\">")
+	addReplacement(REG_BBTAG("/tiny"), "</span>")
 	addReplacement(REG_BBTAG("list"), "<ul>")
 	addReplacement(REG_BBTAG("/list"), "</ul>")
+	addReplacement(REG_BBTAG("agency"),  	"<span style=\"font-family:Agency FB\">")
+	addReplacement(REG_BBTAG("/agency"), 	"</span>")
+	addReplacement(REG_BBTAG("algerian"),  	"<span style=\"font-family:Algerian\">")
+	addReplacement(REG_BBTAG("/algerian"), 	"</span>")
+	addReplacement(REG_BBTAG("arial"),  	"<span style=\"font-family:Arial\">")
+	addReplacement(REG_BBTAG("/arial"), 	"</span>")
+	addReplacement(REG_BBTAG("arialb"),  	"<span style=\"font-family:Arial Black\">")
+	addReplacement(REG_BBTAG("/arialb"), 	"</span>")
```

Comment:
Well, I'm not sure there's a way around that, unfortunately. I can't find any HTML tags that are both differentiable, and don't make a line break after the tag end. In fact <span> was the only tag I could find that did not, in and of itself, change the text inside it in any way by default. Is there any way to differentiate <span> tags from one another?